notebooks/MUS2019_CTF.ipynb

```diff
@@ -0,0 +1,1120 @@
+      ]
+    },
+    {
+      "metadata": {
+        "id": "h35lMbAxIeYE",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Install the TimeSketch API client if you don't have it\n",
+        "!pip install timesketch-api-client\n",
+        "\n",
+        "# Import some things we'll need\n",
+        "from timesketch_api_client import client\n",
+        "import pandas as pd\n",
+        "pd.options.display.max_colwidth = 60"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ph1jGHR5JjZo",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Connect to Timesketch"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "i5JCNPkL87Cq",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "By default, this will connect to the public demo Timesketch server, which [David Cowen](https://twitter.com/HECFBlog) has graciously allowed to host a copy of the Plaso timeline of the MUS2019-CTF. Thanks Dave!"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "dqwwIBOpJfZi",
+        "colab_type": "code",
+        "cellView": "form",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "#@title Client Information { run: \"auto\"}\n",
+        "SERVER = 'https://demo.timesketch.org' #@param {type: \"string\"}\n",
+        "USER = 'demo' #@param {type: \"string\"}\n",
+        "PASSWORD = 'demo' #@param {type: \"string\"}\n",
+        "\n",
+        "ts_client = client.TimesketchApi(SERVER, USER, PASSWORD)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "OZ4CahZZPjht",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Now that we've connected to the Timesketch server, we need to select the Sketch that has the CTF timeline. \n",
+        "\n",
+        "First we'll list the available sketches, then print their names:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "vsEZI45porba",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "sketches = ts_client.list_sketches()\n",
+        "for i, sketch in enumerate(sketches):\n",
+        "  print('[{0:d}] {1:s}'.format(i, sketch.name))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "E6RX0jqzQrRq",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Then we'll select the MUS2019-CTF sketch:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "hvCl6L7ZQrsZ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "ctf = sketches[0]"
```

Comment:
I'm wondering whether something like:

```
ctf = [x for x in sketches if x.name == 'MUSCTF 2019'][0]
```

is better, since you are picking the first sketch in the list... what if another sketch is added to the demo server?

Reply:
I was trying to show how to pick a sketch from the list of sketches, that's why it is two steps. I've added some clarification to the text above, let me know what you think.

Reply:
ack